src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -860,7 +860,7 @@ public function morphTo($name = null, $type = null, $id = null)
         // If the type value is null it is probably safe to assume we're eager loading
         // the relationship. When that is the case we will pass in a dummy query as
         // there are multiple types in the morph and we can't use single queries.
-        if (is_null($class = $this->$type)) {
+        if (is_null($class = $this->$type) || $class === '') {
```

Comment:
Thats definitely a possibility, so an going to dig deeper and determine
what's going on and a fix.
On 28 Apr 2016 19:21, "Graham Campbell" <notifications@github.com> wrote:

> In src/Illuminate/Database/Eloquent/Model.php
> <https://github.com/laravel/framework/pull/13360#discussion_r61477087>:
>
> > @@ -860,7 +860,7 @@ public function morphTo($name = null, $type = null, $id = null)
> >          // If the type value is null it is probably safe to assume we're eager loading
> >          // the relationship. When that is the case we will pass in a dummy query as
> >          // there are multiple types in the morph and we can't use single queries.
> > -        if (is_null($class = $this->$type)) {
> > +        if (is_null($class = $this->$type) || $class === '') {
>
> Maybe this change isn't correct then, and our tests should be failing here
> too but the tests are just incomplete?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/laravel/framework/pull/13360/files/bc1519926bf4a5a541bfb1aafefe0bcba4f0acbe#r61477087>
>